.github/workflows/add-remove-labels.yml

```diff
@@ -31,19 +36,41 @@ jobs:
           comment-id: ${{ github.event.comment.id }}
           reactions: '+1'
 
+      # This currently fails with either the bot PAT or the standard github token secret
```

Comment:
Do you want to remove these?

Reply:
Not really? I'm hoping this actually gets fixed (https://github.com/github/docs/issues/36899) and we can begin using it